docs/SETUP.md

```diff
@@ -0,0 +1,33 @@
+# Setup
+
+This document describes how to setup your local working environment in order to develop Messari subgraphs.
+
+## Prerequisites
+
+- [Node & npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)
+- Some development experience
+- Technical knowledge of blockchains (especially evm)
+- Knowledge of [The Graph](https://thegraph.com/docs/) (our beginner [RESOURCES.md](./RESOURCES.md) file will be a good starting place)
+
+## Installation
+
+After installing Nodejs and npm you are close to done with installing all of the development dependencies. There are a few other npm packages to install globally:
+
+- [graph-cli](https://www.npmjs.com/package/@graphprotocol/graph-cli)
+- [graph-ts](https://www.npmjs.com/package/@graphprotocol/graph-ts)
+- [messari-subgraph-cli](https://www.npmjs.com/package/messari-subgraph-cli) (more on this in [TOOLING.md](./TOOLING.md))
+- [mustache](https://www.npmjs.com/package/mustache)
+
+This is really all that you need, but you can read more about the graph packages [here](https://thegraph.com/docs/en/developing/creating-a-subgraph/#install-the-graph-cli).
+
+After cloning the repo and moving into the head of the repository you should install the project-level npm packages:
+
+```bash
+git clone https://github.com/messari/subgraphs.git
+cd subgraphs
+npm install
+```
```

Comment:
Can maybe add a mention to our git hooks here. When running `npm install` it will also setup `husky` to run some pre-commit hooks we have. Mainly to format the files changed with `prettier`. If this wants to be disabled for some reason it can be done with `husky uninstall`